modules/microshift-rpm-ostree.adoc

```diff
@@ -5,10 +5,12 @@
 :_content-type: PROCEDURE
 [id="microshift-rpm-ostree-package-system_{context}"]
 = rpm-ostree image and package system
+
```

Comment:
I'm a bit confused about this file, which not from my original PR. I don't see it linked from anywhere. Where would we actually add this to the docs?

Reply:
@fzdarsky This is in the Networking section.